docs/HowToFAQ/index.md

```diff
@@ -76,10 +76,24 @@ If you have questions that are not addressed here, be sure to communicate them t
 - ["How do I run boogie manually on the Dafny output?](FAQBoogie)
 - ["Does Dafny verify methods in parallel?"](FAQParallel)
 - ["Is there a doc generator for Dafny?"](FAQDocGenerator)
-- ["Are there any useful tricks (e.g. z3 switches) to get better automation for nonlinear arithmetic (perhaps trading off other performance aspects)?"](FAQNonlinearArith)
+- ["How can I improve automation and performance for programs with non-linear arithmetic?"](FAQNonlinearArith)
 - ["It looks like, when compiling to C#, my print statements don't show up if I don't have \n at the end of the string."](FAQNewline)
 - ["Is there a standard library for Dafny?"](FAQStandardLibrary)
 - ["Why do I need to use an old Z3?"](FAQZ3)
 - ["How do I ask a question or file a problem report or make a suggestion about Dafny?"](FAQIssues)
 - ["Any plans to release the language server as a NuGet package? Seems like it’s not part of the Dafny release."](FAQNuget)
 - ["What compiler target languages are in development?"](FAQCompileTargets)
+
+# Errors
+
+- ["'z3' cannot be opened because the developer cannot be verified."](ERROR_Z3)
+- ["rbrace expected"](ERROR_Rbrace)
+- ["closeparen expected"](ERROR_CloseParen)
+- ["cannot establish the existence of LHS values that satisfy the such-that predicate"](ERROR_NoLHS)
+- ["type parameter 0 (K) passed to type A must support no references"](ERROR_NoReferenceTypeParameter)
+- ["a modifies-clause expression must denote an object or a set/iset/multiset/seq of objects (instead got map<int, A>)"](ERROR_ModifiesValue)
+- ["name of datatype (String) is used as a function"](ERROR_DataTypeName)
+- ["possible violation of function precondition for op(v)"](ERROR_FunctionPrecondition)
+- ["type ? does not have a member IsFailure"](ERROR_IsFailure)
+- ["value does not satisfy subset constraints of ?"](ERROR_Covariance)
```

Comment:
Are these entries supposed to be in this PR?

Reply:
Those are already in master -- so I'm suspect of the diff.